resources/views/pages/game/[game]/comments.blade.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+use App\Models\Game;
+use Illuminate\View\View;
+
+use function Laravel\Folio\{middleware, name, render};
+
+middleware(['auth', 'can:viewAny,' . Game::class]);
```

Comment:
Recommend removing "auth" from this (and any other) public-facing comments routes.

This is a subtle behavior change, but it negatively impacts SEO. All of a sudden our most popular game pages lose a lot of unique linked content because they're no longer crawlable.

Reply:
done